cg/server/admin.py

```diff
@@ -441,6 +447,68 @@ class AnalysisView(BaseView):
     form_extra_fields = {"workflow": SelectEnumField(enum_class=Workflow)}
 
 
+class IlluminaFlowCellView(BaseView):
+    """Admin view for Model.IlluminaSequencingRun"""
+
+    column_list = (
+        "internal_id",
+        "sequencer_type",
+        "sequencer_name",
+        "sequenced_at",
+        "data_availability",
+        "has_backup",
+        "total_reads",
+        "total_undetermined_reads",
+        "percent_q30",
+        "mean_quality_score",
+        "total_yield",
+        "yield_q30",
+        "cycles",
+        "demultiplexing_software",
+        "demultiplexing_software_version",
+        "sequencing_started_at",
+        "sequencing_completed_at",
+        "demultiplexing_started_at",
+        "demultiplexing_completed_at",
+        "archived_at",
+    )
+    column_formatters = {
+        "internal_id": view_flow_cell_internal_id,
+    }
+
+    column_default_sort = ("sequenced_at", True)
+    column_filters = ["sequencer_type", "sequencer_name", "data_availability"]
+    column_editable_list = ["data_availability"]
+    column_searchable_list = ["sequencer_type", "sequencer_name", "device.internal_id"]
+    column_sortable_list = [
+        "sequenced_at",
+        "sequencer_type",
+        "sequencer_name",
+        "data_availability",
+        "has_backup",
+        "sequencing_started_at",
+        "sequencing_completed_at",
+        "demultiplexing_started_at",
+        "demultiplexing_completed_at",
```

Comment:
Once I know if we can get accurate dates here we can keep this..

Also, if we have these 4 time stamps. What purpose does the `sequenced_at` serve?

Reply:
fixed in source branch